pyproject.toml

```diff
@@ -85,6 +85,8 @@ torch = ">=1.8.0"
 torchmetrics = ">=0.6.0"
 tqdm = ">=4.56.0"
 typing_extensions = {version = "*", python = "<3.8", optional = true}
+huggingface_hub = "*"
```

Comment:
can we make this an optional dependency? @martinkim0 had a good solution for the autotune PR

Reply:
why would we make it an optional dependency? @adamgayoso

Reply:
this dependency should not stop someone from using the core functionality of this package.